src/frontend/komponenter/Fagsak/Saksoversikt/Saksoversikt.tsx

```diff
@@ -109,7 +116,7 @@ const Saksoversikt: React.FunctionComponent<IProps> = ({ minimalFagsak }) => {
             <Lenke
                 href={`/fagsak/${minimalFagsak.id}/${aktivBehandling.behandlingId}/tilkjent-ytelse`}
             >
-                Se behandlingsresultat for detaljer
+                Se detaljer
```

Comment:
Tok meg friheten til å endre teksten her. Med den nye alert-komponenten ble denne lenken så lang at den ble fordelt på to linjer, noe som så veldig rart ut. Føler denne teksten får frem budskapet minst like bra som tidligere 😊 rop ut om noen (Halvor 😝) er uenige!

Reply:
Ser bra ut det 👍